lib/car.rb

```diff
@@ -0,0 +1,31 @@
+class Car
+  attr_accessor :make_model, :monthly_payment, :loan_length, :color
+
+  def initialize(make_model, monthly_payment, loan_length)
+    @make_model = make_model
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    @make_model = make_model
+    @color
+  end
+
+  def make
+    make = []
+    make = @make_model.split(" ")
+    make[0]
+  end
```

Comment:
You don't need to set make=[] here since you are immediately overwriting it on the next line. With your approach this variable naming may make more sense:

```
def make
   make_model_array = @make_model.split(" ") #since at this stage it is still both the make and model
   make_model_array[0]
end
```